### PR TITLE
feat(LandingPage): add landing page complex investigation index

### PR DESCRIPTION
**Pull request by RoeiRom**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -69,45 +73,45 @@ const InvestigationTable: React.FC = (): JSX.Element => {
 
     const user = useSelector<StoreStateType, User>(state => state.user);
 
-    const CustomPopper = (props: any) => {
-        return (<Popper {...props} className={classes.popperStyle} placement='bottom-start' />)
-    }
+    // const CustomPopper = (props: any) => {
+    //     return (<Popper {...props} className={classes.popperStyle} placement='bottom-start' />)
+    // }
```
Comment (nevobonneh) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:78]: delete comment

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -69,45 +73,45 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
 
     const getTableCell = (cellName: string, indexedRow: { [T in keyof typeof TableHeadersNames]: any }) => {
         switch (cellName) {
             case TableHeadersNames.investigatorName:
                 if (selectedRow === indexedRow.epidemiologyNumber && investigatorAutoCompleteClicked) {
                     return (
                         <Autocomplete
-                            PopperComponent={CustomPopper}
+                            // PopperComponent={CustomPopper}
```
Comment (nevobonneh) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:86]: delete comment

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 3]
```diff
@@ -69,45 +73,45 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+                            //                 {activeInvestigationsMsg} :
+                            //                     <b>
+                            //                         {option.value.activeInvestigationsCount}
+                            //                     </b>
+                            //             </Typography>
+                            //         </div>
+                            //     </>
+                            //     :
+                            //     ''
+                            // )}
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:114]: Delete comment

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 4]
```diff
@@ -125,9 +129,9 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                     placeholder='חוקר'
                                 />
                             }
-                            classes={{
-                                option: classes.userSelectOption
-                            }}
+                            // classes={{
+                            //     option: classes.userSelectOption
+                            // }}
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:134]: Delete comment

**Review by MayaSasson — APPROVED** [review]

**Review by yaniryos — APPROVED** [review]

**Pull request merged**